scipy/spatial/transform/rotation.pyx

```diff
@@ -1012,14 +1015,110 @@ cdef class Rotation(object):
         quat = _elementary_quat_compose(seq.encode(), angles, intrinsic)
         return cls(quat[0] if is_single else quat, normalize=False, copy=False)
 
+    @classmethod
+    @cython.boundscheck(False)
+    @cython.wraparound(False)
+    def from_mrp(cls, mrp):
+        """Initialize from Modified Rodrigues Parameters (MRPs).
+
+        MRPs are a 3 element attitude description commonly used in the aerospace
+        industry.
+
+        Parameters
+        ----------
+        mrp : array_like, shape (N, 3) or (3,)
+            A single vector or a stack of vectors, where `mrp[i]` gives
+            the ith set of MRPs.
+
+        Returns
+        -------
+        rotation : `Rotation` instance
+            Object containing the rotations represented by input rotation
+            vectors.
+
+        References
+        ----------
+        .. [1] Shuster, M. D. "A Survery of Attitude Representations",
+               The Journal of Astronautical Sciences, Vol. 41, No.4, 1993,
+               pp. 475-476
+
+        Notes
+        -----
+
+        .. versionadded:: 1.5.3
```

Comment:
Just FYI -- this will probably have to be `1.6.0` if the feature is approved because we have already released the `1.5.x` series and almost always restrict point releases to bug fixes rather than enhancements/new features.

Reply:
Updated